modules/rhdh-install-web-console.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * layered-products-web-console.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rhdh-install-web-console_{context}"]
+=  Installing the {rh-dev-hub} using the {product-title} web console
+
+The web console provides instructions in a quick start to install the {rh-dev-hub} Operator.
```

Comment:
I think this might be misread as "start to install", so I would reformulate as something like "The web console provides a quick start with instructions about installing..."